custom/echis_reports/ucr/data_sources/household_characteristics_data_source.json

```diff
@@ -0,0 +1,501 @@
+{
+   "domain":"fmoh-echis-staging",
+   "es_index_settings":{
```

Comment:
This is not a valid field, where did you get this from? Can you point me to the UCR source on echisethipia.org?

Reply:
I used the json after clicking the button "Data source source(Advanced)".

Reply:
just sent you the link on slack private message

Reply:
@andyasne This comment is still not addressed.